tools/bumbershoot/custom_pro_db/tool-data/customProDB.loc.sample

```diff
@@ -1,18 +0,0 @@
-#This file lists the locations and dbkeys of all the fasta files
-#under the "genome" directory (a directory that contains a directory
```

Comment:
I do use the data table, but it's created by the data_manager. Planemo broke when I deleted `tool_data_table_conf.xml.sample`, but it still works with the tool-data directory deleted. Is there another reason to keep it?